connectors/sources/s3.py

```diff
@@ -105,7 +111,7 @@ async def get_bucket_list(self):
             buckets = self.configuration["buckets"]
         return buckets
 
-    async def get_bucket_objects(self, bucket):
+    async def get_bucket_objects(self, **kwargs):
```

Comment:
Sorry didn't realize it in the first review. I think this change makes the `bucket` parameter optional.

What do you think if we change it to 
```suggestion
    async def get_bucket_objects(self, bucket, prefix=None):
```

or 

```suggestion
    async def get_bucket_objects(self, bucket, **filters):
```